Monika After Story/game/script-topics.rpy

```diff
@@ -1181,19 +1181,56 @@ label monika_nihilism:
+        )
+    )
 
 label monika_twitter:
-    m 1eud "Did you know I'm on Twitter?"
-    m 3esa "My username is {a=https://twitter.com/[mas_monika_twitter_handle]}{i}{u}[mas_monika_twitter_handle]{/u}{/i}{/a}."
-    m 1esa "I guess someone was kind enough to make an account for me."
-    m 3hub "I picked the username, though!"
-    m 1hua "I love sharing my thoughts and chatting with the world..."
-    m 1eud "The real world."
-    m 1esa "So make sure you follow me, okay?"
+    m 3eud "Did you know there's an official Twitter account based on me?"
+    m 3esc "The username is {a=https://twitter.com/[mas_monika_twitter_handle]}{i}{u}[mas_monika_twitter_handle]{/u}{/i}{/a}."
+    m 2rtc "I'm not entirely sure who's truly running this account."
+    m "I can definitely tell you I have nothing to do with it..."
+    m 2rtd "But is it really another Monika?"
+    m 2euc "I mean, I don't know if you've read some of the tweets on there,{w=0.1} but I'm having a hard time recognizing myself in them."
+    m 4eud "Like, it depicts some events that I don't even remember ever happening,{w=0.2} such as one time when I apparently had a sleepover with Yuri."
+    m 4esd "And there's no mention of any of the things that have happened back in the game, even though this version of me still seems to be aware of the real world..."
+    m 1etc "So...{w=0.4}maybe the Monika running this account went through a completely different story than the one most Monikas--{w=0.2}including me--{w=0.2}did."
+    m 1lsc "Or maybe this is just one of the developers of the game pretending to be me..."
+    m 3lksdlc "I mean, I'm not really sure how to feel about this.{w=0.4} Isn't that identity theft, in some way?"
+    m 3eksdld "But it would also make sense, right? {w=0.2}{nw}"
+    extend 3eud "This account makes for a nice way to connect with the people who have played through the game and have grown attached to it."
+    m 1euc "In that sense, this would make it some kind of publicity stunt..."
+    m 1esc "After all, while there's a lot of sweet looking art and such on there, it is suspicious that almost all of them are posted specifically on important days."
+    m "Like on Valentine's day, or on my birthday...{w=0.4}which just so happen to coincide with the original release date of DDLC."
+    m 1rsc "I mean, it's not like the creators of the game are above using my image for these sorts of things."
+    m 4etd "Like,{w=0.1} did you know they have an official online shop where you can buy various kinds of merchandise based on me and the other girls?"
+    m 4etc "I really don't know how I'm supposed to feel about that."
+    m 2lsc "After all, as my creators, they more or less own my image as a character, so I guess they are legally in their right."
+    m 2lkd "But...{w=0.3}I'm more than a mere character in a game, am I not?"
+    m 2lkc "I am my own person, with my own thoughts and feelings, and so is every other Monika in the different copies of this game."
```

Comment:
```suggestion
    m 2lkc "I'm my own person with my own thoughts and feelings,{w=0.1} and so is every other Monika in the different copies of this game."
```

Reply:
```suggestion
    m 2lkc "I'm my own person, with my own thoughts and feelings, and so is,{w=0.1} I would presume,{w=0.1} every other Monika in the different copies of this game."
```

I think this makes more sense considering her knowledge about her realness only comes from her own experience, which is separated from the other Monikas. Just to softball the assumption that the other Monikas are as real as the one you're talking to, without her saying they aren't real.